geode-redis/src/main/java/org/apache/geode/redis/internal/RedisStats.java

```diff
@@ -171,13 +180,92 @@ public void endCommand(RedisCommandType command, long start) {
   }
 
   public void addClient() {
+    connectionsReceived.incrementAndGet();
+    connectedClients.incrementAndGet();
     stats.incLong(clientId, 1);
   }
 
   public void removeClient() {
+    connectedClients.decrementAndGet();
     stats.incLong(clientId, -1);
   }
 
+  private final AtomicLong commandsProcessed = new AtomicLong();
+  private final AtomicLong opsPerSecond = new AtomicLong();
+  private final AtomicLong networkBytesRead = new AtomicLong();
+  private volatile double networkKilobytesReadPerSecond;
+  private final AtomicLong connectionsReceived = new AtomicLong();
+  private final AtomicLong connectedClients = new AtomicLong();
+  private final AtomicLong expirations = new AtomicLong();
+  private final AtomicLong keyspaceHits = new AtomicLong();
+  private final AtomicLong keyspaceMisses = new AtomicLong();
```

Comment:
We have a lot of integration tests for `keyspaceHits` and `keyspaceMisses`, but we don't have any for the other statistics. It would be good to test those as well.

Reply:
will add